doc/experiment.rst

```diff
@@ -1,327 +1,454 @@
+  If you would like to launch the Ensemble as a batch job and
+  define a set of parameter values
+  for which you want to test different combinations/permutations,
+  initialize an ``Ensemble`` with a ``BatchSettings``, ``RunSettings`` objects
+  and `params` argument. The run settings and parameters will
+  expand to all Ensemble members. The Ensemble will be launched as
+  a single batch job.
+
+Case 5 : Launch as a batch job and define number of Models created.
+  If you would like to launch the Ensemble as a batch job
+  manually define the number of models created
+  within an Ensemble, initialize an ``Ensemble`` with a ``RunSettings`` object and
+  `replicas` argument. The `replicas` argument will create clones
+  of the ``Model`` object. The run settings will
+  expand to all Ensemble members.
+
+The ``create_ensemble()`` factory method returns an initialized ``Ensemble`` object that
+gives you access to functions associated with the :ref:`Ensemble API<ensemble_api>`.
+
+.. note::
+  For more information on all cases and input arguments, navigate to the Ensemble documentation page.
+==================
+Experiment Example
+==================
+.. compound::
+  In the following subsections, we provide an example of using SmartSim to automate the
+  deployment of an HPC workload and distributed, in-memory storage, within
+  the workflow.
+
+  Continue to the example to:
+
+  .. list-table:: Experiment example contents
+   :widths: auto
+   :header-rows: 1
+
+   * - Initialize
+     - Start
+     - Stop
+   * - a workflow (``Experiment``)
+     - the in-memory database (``Orchestrator``)
+     - the in-memory database (``Orchestrator``)
+   * - a in-memory database (``Orchestrator``)
+     - the workload (``Model``)
+     - 
+   * - a workload (``Model``)
+     - 
+     - 
+
+Initialize
+----------
+.. compound::
+  To create a workflow, we *initialize* an ``Experiment`` object
+  once at the beginning of the Python driver script.
+  To create an Experiment, we specify a name
+  and the system launcher with which we will execute the driver script.
+  Here, we are running the example on a Slurm machine and as such will
+  set the `launcher` argument to `slurm`.
+
+  .. code-block:: python
+
+      from smartsim import Experiment
+      from smartsim.log import get_logger
+
+      # Initialize an Experiment
+      exp = Experiment("name-of-experiment", launcher="slurm")
+      # Initialize a SmartSim logger
+      smartsim_logger = get_logger("tutorial-experiment")
+
+  We also initialize a SmartSim logger. We will use the logger throughout the experiment
+  to monitor the entities.
+
+.. compound::
+  Next, we will launch a SmartSim in-memory database called an ``Orchestrator``.
+  To *initialize* an ``Orchestrator`` object, use the ``Experiment.create_database()``
+  function. We will create a single-sharded database and therefore will set
+  the argument `db_nodes` to 1. SmartSim will assign a `port` to the database
```

Comment:
"SmartSim will assign a `port` to the database
  and detect your machine's `interface`." -> 
  SmartSim will assign a `port` to the database
  and attempt to detect your machine's `interface` if values are not provided to the ``Experiment.create_database()`` factory method."